### ISSUE TITLE
Sourcekitten isn't picking up deprecation attributes on APIs

### ISSUE THREAD

**Issue by jverkoey** [material-motion/apidiff#10]
This is a pretty critical omission from the generated release notes unfortunately.

https://github.com/mattstevens/objc-diff was able to pick up deprecation attributes, so we may want to look how they did so.

**Comment by jverkoey** [comment]
Where objc-diff is exposing availability: https://github.com/mattstevens/objc-diff/blob/95998186066dc66e50e291605d011d8b6a4d4d52/Dependencies/ObjectDoc.framework/Versions/A/Headers/PLClangAvailability.h

**Comment by jverkoey** [comment]
Related open bug in sourcekitten: https://github.com/jpsim/SourceKitten/issues/214

**Comment by jverkoey** [comment]
This appears to be where availability is being extracted https://github.com/landonf/objectdoc/blob/dc31c31dce677040362f77a2c17c5e0089838c1b/ObjectDoc/PLClangAvailability.m

**Comment by jverkoey** [comment]
Seems this API is available in sourcekitten: https://github.com/jpsim/SourceKitten/blob/42b70b0e281ac9d72e436b8b1a4ce9f268102f0d/Source/SourceKittenFramework/library_wrapper_Index.swift#L88

**Comment by jverkoey** [comment]
Have confirmed that I can extract availability information from a cursor in objective-c code with the following snippet:

```

    func platformAvailability() {
        var always_deprecated: Int32 = 0
        var always_unavailable: Int32 = 0
        var deprecationString = CXString()
        var unavilableString = CXString()

        let platformCount = clang_getCursorPlatformAvailability(self, nil, nil, nil, nil, nil, 0);
        clang_getCursorPlatformAvailability(self,
                                            &always_deprecated,
                                            &deprecationString,
                                            &always_unavailable,
                                            &unavilableString,
                                            nil,
                                            platformCount)
        print("\(always_deprecated) \(deprecationString)")
    }
```

**Comment by jverkoey** [comment]
I have a working diff that includes deprecation status inbound.

**Comment by jverkoey** [comment]
Verified that apidiff picks up the new keys and generates deprecation change notices.

**Comment by jverkoey** [comment]
This is resolved in [SourceKitten 0.14.1](https://github.com/jpsim/SourceKitten/releases/tag/0.14.1).
